Cargo.toml

```diff
@@ -142,7 +142,7 @@ loki-logproto = { path = "lib/loki-logproto", optional = true }
 async-stream = { version = "0.3.5", default-features = false }
 async-trait = { version = "0.1.68", default-features = false }
 futures = { version = "0.3.28", default-features = false, features = ["compat", "io-compat"], package = "futures" }
-tokio = { version = "1.26.0", default-features = false, features = ["full"] }
+tokio = { version = "1.27.0", default-features = false, features = ["full"] }
```

Comment:
This was necessary to get access to impls of [`std::os::fd::AsFd`](https://doc.rust-lang.org/stable/std/os/fd/trait.AsFd.html) for the various socket types in `tokio`, which is required by `socket2` when creating a `SockRef` around a socket.

That code is the stuff in `src/net.rs` that makes setting send/receive buffer sizes, etc, generic over the socket type (TCP, UDP, or Unix).

Reply:
Admittedly, I may have missed another way to do it that doesn't involve updating `tokio`, but I think it would also then involve downgrading `socket2` to `0.4.x`?

Reply:
Irrelevant now as `master` has moved on to `1.28.1`.